setup.cfg

```diff
@@ -28,6 +28,12 @@ classifiers =
 [bdist_wheel]
 universal = 1
 
+[mypy]
+warn_unused_configs = True
+pretty = True
+files = src
+check_untyped_defs = True
```

Comment:
Can these live in `pyproject.toml`?

Reply:
PyTest 6.0 is adding pyproject.toml support. Black I believe always worked either place, IIRC.

Reply:
I suppose you like [PEP 621](https://www.python.org/dev/peps/pep-0621) then :)